### PR TITLE
[stable/coredns] Update deprecated api version

### PR DESCRIPTION
**Pull request by alexppg**
Signed-off-by: Alex Perez-Pujol <alexperezpujol@disroot.org>

#### What this PR does / why we need it:
Fixes PSP's api group, since it was deprectated.
https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by mrueg** [comment]
Title should be [stable/coredns]

**Comment by alexppg** [comment]
Changed.

File: stable/coredns/templates/podsecuritypolicy.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{- if .Values.rbac.pspEnable }}
-apiVersion: extensions/v1beta1
+apiVersion: policy/v1beta1
```
Comment (davidkarlsen) [inline stable/coredns/templates/podsecuritypolicy.yaml:5]: by using capabilities: https://helm.sh/docs/topics/chart_template_guide/builtin_objects/ the deployment can work on newer and older clusters.

Comment (alexppg) [reply stable/coredns/templates/podsecuritypolicy.yaml:5]: Cool, I didn't know about capabilities, I just added them.

**Comment by mrueg** [comment]
> Changed.

should be coredns not coreos :-)

**Comment by alexppg** [comment]
Ugh, sorry, it seems I wasn't paying atention.

**Pull request merged**